src/components/ObsDetails/ActivityItem.js

```diff
@@ -48,6 +52,18 @@ const ActivityItem = ( {
     isActiveUserTheCurrentUser( );
   }, [user] );
 
+  const handleSuccess = {
+    onSuccess: ( ) => {
+      queryClient.invalidateQueries( ["fetchRemoteObservation", item.uuid] );
+      refetchRemoteObservation( );
+    }
+  };
+
+  const deleteCommentMutation = useAuthenticatedMutation(
+    ( uuid, optsWithAuth ) => deleteComments( uuid, optsWithAuth ),
+    handleSuccess
```

Comment:
`useAuthenticatedMutation` takes three arguments: `queryFunction`, `handleCallback`, and `queryOptions`. As far as I can tell, the only reason this might be working is that you're basically treating the second arg (`handleCallback`) as if it were an object literal of options you pass to `useMutation`, so...

1. Please rename the second argument for `useAuthenticatedMutation` so it describes what it is, maybe something like `mutationOptions`
2. Why isn't flow complaining about this? The arg is declared as `handleCallback: Function`, and yet right here you're passing it an object literal. Shouldn't flow catch that and complain?